spec/support/tests/filesystem.rb

```diff
@@ -63,9 +63,9 @@ def create_parents_of(path)
       wrap(path).dirname.mkpath
     end
 
-    def append_to_file(path, content, options = {})
+    def append_to_file(path, content, _options = {})
       create_parents_of(path)
-      open(path, 'a') { |f| f.puts(content + "\n") }
```

Comment:
The complain was from [Security/Open](https://docs.rubocop.org/rubocop/cops_security.html#securityopen). I see that there is an open method, maybe rubocop is to blame here. Will check with rubocop and fix there in case found guilty!